src/store/actions/settings.js

```diff
@@ -10,7 +10,16 @@ export const settingsRetrieved = () => (dispatch) => {
   getFromStorage('settings', initialState, (data) => {
     dispatch({
       type: actionTypes.settingsRetrieved,
-      data,
+      data: {
+        ...data,
+        token: {
+          active: 'LSK',
```

Comment:
Same. I would prefer all the 'LSK' strings replaced by `tokenMap.LSK.key`

Reply:
Fixed